### PR TITLE
Feature/dashboard graph

### LINKED ISSUE #833: Dashboard graph
**[Zeplin](https://app.zeplin.io/project/5e4e7717ae5dfd7c427a5a28/screen/5e79c9bc8b52437106eac048)**

### LINKED ISSUE #1074: Wallet graph (v.1)
- [ ] Graph [shown](https://sketch.cloud/s/AK1WY/a/WOMWGg) / [hidden](https://sketch.cloud/s/AK1WY/a/3gKdwD)
- [ ] Tooltip on hover: _Rate updated (time) ago_
- [ ] Styling 

Zeplin **[here](https://app.zeplin.io/project/5e4e7717ae5dfd7c427a5a28/screen/5e79c9bc8b52437106eac048)**

### LINKED ISSUE #1473: Transactions: state update on unmounted component
Screenshot 2020-04-07 at 19 58 26

### PR DESCRIPTION
**Pull request by slowbackspace**
close https://github.com/trezor/trezor-suite/issues/833
close https://github.com/trezor/trezor-suite/issues/1473
close https://github.com/trezor/trezor-suite/issues/1074#issuecomment-607308784

not cached to storage for now as I haven't mplemented a strategy to automatically refresh stale graph data yet

### CODE REVIEW

File: packages/suite/src/components/dashboard/PortfolioCard/components/DashboardGraph/index.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                        Could not load data{' '}
+                        <Button
+                            onClick={() => {
+                                props.updateGraphData(accounts, selectedRange);
+                            }}
+                            icon="REFRESH"
+                            variant="tertiary"
+                            size="small"
+                        >
+                            Retry
```
Comment (mroz22) [inline packages/suite/src/components/dashboard/PortfolioCard/components/DashboardGraph/index.tsx]: `<Translation id="TR_RETRY" />`

Comment (mroz22) [reply packages/suite/src/components/dashboard/PortfolioCard/components/DashboardGraph/index.tsx:88]: <Translation id="TR_RETRY" />

File: packages/suite/src/components/suite/FormattedNumber/index.tsx [hunk 2]
```diff
@@ -26,8 +26,8 @@ export default (props: Props) => {
         <FormattedNumber
             currency={props.currency}
             value={bn.toNumber()}
-            minimumFractionDigits={props.minimumFractionDigits || 2}
-            maximumFractionDigits={props.minimumFractionDigits || 2}
+            minimumFractionDigits={props.minimumFractionDigits ?? 2}
```
Comment (mroz22) [inline packages/suite/src/components/suite/FormattedNumber/index.tsx:29]: nice! (??) :D

Comment (slowbackspace) [reply packages/suite/src/components/suite/FormattedNumber/index.tsx:29]: yeah, I spent an ungodly amount of time wondering why passing '0' doesn't work

**Pull request merged**